### DISCUSSION TITLE
Adjust Opacity of Watermark Text

### DISCUSSION THREAD

**Discussion by aamirencodes** [ImageMagick/ImageMagick#3570] [Help]
Using the below command I'm trying to watermark images using ImageMagick, I'm unable to adjust the opacity of the text "Sample Text" with which the image is getting watermarked with. Can you please help with how that can be done please?

`magick "Source_Image_location.JPG" -pointsize 50 -gravity center -family "ComedySansCondensed-Black" -fill "White" -draw " rotate 45 text 0,0 'SAMPLE TEXT'" -resize 360x500\! -strip "Dest_Image_location.JPG"`

Thanks in Advance!

**Comment by fmw42** [comment]
Change your -fill color from "white" to "rgba(255,255,255,0.5)", which is half transparent white

**Reply by aamirencodes** [reply]
Thank you for helping on this @fmw42 , this works!
